Rebuild with libwebp 1.1 (latest pin)

xylar opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

xylar committed:
Bump build number

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.


I do have some suggestions for making it better though...


For **recipe**:

 * License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.

Documentation on acceptable licenses can be found [here]( https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions ).

xylar commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 3.20.2, conda-smithy 3.7.10, and conda-forge-pinning 2020.09.26.15.06.30

xylar committed:
Fix license name

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

xylar commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 3.20.2, conda-smithy 3.7.10, and conda-forge-pinning 2020.09.26.15.06.30

xylar commented:
The pin for `libwebp` got updated here: 
https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/553 
but apparently no migrator was created so we need to do it manually.

xylar commented:
@CurtLH, I don't use this package so I wasn't going to add myself as a maintainer but if you could use a hand keeping this updated, let me know.

xylar committed:
Fix broken patch

There was some extraneous whitespace and text that seems to be
confusing `patch`.

xylar commented:
@CurtLH, this is ready to review when you get a chance.  This would need to be merged for a related fix for `tesseract` to be possible: https://github.com/conda-forge/tesseract-feedstock/pull/24

CurtLH commented:
@xylar -- thanks for the PR.  I also don't currently use this package, so if you'd like to be a maintainer, feel free.  Otherwise, I'm happy to merge this as-is.

xylar committed:
Add xylar as a (somewhat reluctant) maintainer

xylar commented:
@conda-forge-admin, please rerender

xylar commented:
@CurtLH, okay, sounds fair.  I'll add myself as a maintainer and we can see how it goes.  I can try to clear the backlog of unmerged PRs as I have some time.

Please feel free to merge as soon as tests have passed (or set the automerge flag if you prefer).

Commit:
MNT: Re-rendered with conda-build 3.20.2, conda-smithy 3.7.10, and conda-forge-pinning 2020.09.28.09.54.05

xylar commented:
Thanks again!
